test/geocoding_test.rb

```diff
@@ -66,5 +70,13 @@ class GeocodingTest < Test::Unit::TestCase
       end
     end
 
+    should "#geocode_forward (set request and open timeouts)" do
+      Mapbox.request_timeout = 10
+      Mapbox.request_open_timeout = 5
+      result = Mapbox::Geocoder.geocode_forward("Chester, NJ", {:proximity => {:longitude => 0.1234567, :latitude => -10.987654}})
+      assert result
```

Comment:
Same thing - we are interested only in timeouts so no need to verify the result (such check already exists). So maybe just delete this row and change title to something like `should "set_custom_timeouts"`?